src/components/WalletConnect/NanoContract/NewNanoContractTransactionRequest.js

```diff
@@ -155,6 +157,7 @@ export const NewNanoContractTransactionRequest = ({ ncTxRequest }) => {
 
   const onFeedbackModalDismiss = () => {
     dispatch(setNewNanoContractStatusReady());
+    dispatch(newNanoContactRetryDismiss());
```

Comment:
Why do you need to set retry dismiss here?

Reply:
Refactored it to run this in the unmount of the component